### DISCUSSION TITLE
Should the download of SD v1.4 be included in upcoming versions of this repo?

### DISCUSSION THREAD

**Discussion by AbdBarho** [AbdBarho/stable-diffusion-webui-docker#203] [Polls]
All of the UIs in this repo now support SD v1.5! which is automatically downloaded alongside v1.4.

Removing the v1.4 will save roughly 4GB of download for new users, and anyone who wants to use the old version can still do so, but they have to download the model manually.

**Comment by DevilaN** [comment]
There are other things to consider. What about inpainting checkpoint which is like a breakthrough for in- out-painting?

I would like to see interactive download. This could be done with `dialog` with multiple options and default ones checked, so user could decide.
![obraz](https://user-images.githubusercontent.com/12526613/200259277-2ab54906-2cb6-47a9-ba01-387219fddfd3.png)

**Reply by AbdBarho** [reply]
I agree with inpainting model, I will add it to my todo list

I am unsure about the optional ones, some uis require configuration and expect the models to be there ([example](https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/master/services/lstein/models.yaml)) we would then have to generate the config on the fly, which won't be fun.

**Comment by dathide** [comment]
Since it is so easy to move ckpts to the necessary folder, I think only the latest version of SD should be default. 1.5 is also just better in every way, unless I'm missing something.
